apps/etterlatte-pdltjenester/src/main/kotlin/pdl/mapper/PersonMapper.kt

```diff
@@ -66,13 +75,24 @@ object PersonMapper {
                     },
                 kontaktadresse = hentPerson.kontaktadresse?.let { AdresseMapper.mapKontaktadresse(ppsKlient, it) },
                 statsborgerskap = statsborgerskap?.land,
+                pdlStatsborgerskap = pdlStatsborgerskap,
```

Comment:
Mulig `statsborgerskapMedHistorikk` er mer forklarende for denne da begge de statsborgerskap-parameterene der strengt tatt kommer fra pdl?

Reply:
Egentlig ville jeg nok ha renamet det originale feltet til å være "avklartStatsborgerskap" og denne til noe ala "alleStatsborgerskap" heller, men det er værre å rename det gamle feltet